src/debugger/launch.ts

```diff
@@ -128,7 +131,11 @@ export function createTestConfiguration(
         folder = `${workspaceFolder}}/${ctx.relativePath}`;
         nameSuffix = ` (${ctx.relativePath})`;
     }
-    const testEnv = configuration.testEnvironmentVariables;
+    // respect user configuration if conflicts with injected runtime path
+    const testEnv = {
+        ...swiftRuntimePathEnv(),
+        ...configuration.testEnvironmentVariables,
```

Comment:
Because `configuration.testEnvironmentVariables` is last its variables will take precedence in any variable name clashes. Do you want that?

Reply:
Yes, it is left intentionally. We should always allow the user to overwrite the path if they wish.

Reply:
I don’t know if this is the best way, but the reverse should be absolutely wrong👀 Users cannot change runtime path settings easily because it is the milestone of all Swift tools, so if it has the highest priority the users will lose the control to test environment.

Reply:
ok, that's cool